src/lib/emailMgrV2.js

```diff
@@ -0,0 +1,156 @@
+const nacl = require('tweetnacl')
+nacl.util = require('tweetnacl-util')
+const resolve = require('did-resolver').default
+const registerResolver = require('muport-did-resolver')
+const Multihash = require('multihashes')
+const sha256 = require('js-sha256').sha256
+const EmailMgr = require('./emailMgr')
+const { RedisStore } = require('./store')
+const IPFS = require('ipfs-mini')
+
+const IPFS_NODE = new IPFS({ host: 'ipfs.infura.io', port: 5001, protocol: 'https' })
+registerResolver(IPFS_NODE)
```

Comment:
We don't actually need to pass an ipfs instance here. That's only needed for alpha.3, sorry for the confusion. Also ipfs-mini is not compatible with the standard ipfs api so it wouldn't work if we where using alpha.3 ;)

Reply:
My bad, I relied on the `^` prefix for package versions, fixed!